src/components/comment/AddComment.jsx

```diff
@@ -35,6 +36,7 @@ function AddComment({
 			<form onSubmit={e => handleSubmit(e)}>
 				<textarea
 					required
+                    key="comment"
 					id="comment"
```

Comment:
Better with the use context, however this textarea still isnt a controlled component. 
It need a value property, set to the comment state. 
That way, react is in control of setting the value of the textarea i.e. it controls it.